.gitignore

```diff
@@ -129,4 +129,5 @@ dmypy.json
 .pyre/
 
 .idea/
-tests/fixtures/marine/libmarine.so
\ No newline at end of file
+tests/fixtures/marine/libmarine.so
+tests/marine/benchmark/libmarine.so
```

Comment:
Doesn't it get annoying to have so many copies of this `.so` in our tests?

Reply:
Yeah